### PR TITLE
transparent proxy: initial documentation for Local CLI

### PR DESCRIPTION
**Pull request by LouManglass**
Would you mind giving this a once-over? I've viewed it locally for a visual check, which looks fine to me. My primary concern is the content reads well for the CLI transparent proxy mode.

I also made changes to nav_order, you can safely ignore those as they're just a TOC shuffle.

This is just intended for the proxy feature on the local CLI.

@JaapRood FYI documentation is under review for Local CLI.

### CODE REVIEW

File: docs/captures/captures.markdown [hunk 1]
```diff
@@ -31,11 +31,48 @@ tasks:
...
+
+```
+
+*Note:* if not specified the targetUrl will assume ports based on the protocol (80 for http, 443 for https). It may be optionally omitted in these cases. 
+
+Next, run `api check start-proxy`. This will validate that the configuration is correct. Once the configuration check passes, you can [document your API](/documenting-your-api) with `api run start-proxy`. 
+
+- The local dashboard will be available, with observed diffs.
+- The Optic CLI will listen on localhost at the port defined in `inboundUrl`.
+- Traffic sent to the Optic CLI will be forwarded to the defined `targetUrl` - assure that is already running, or if not, start it before sending any traffic.
```
Comment (trulyronak) [inline docs/captures/captures.markdown:70]: these variables are also accessible by env vars, unsure if this is stated elsewhere in the docs

(https://github.com/opticdev/optic/pull/233)

Comment (JaapRood) [reply docs/captures/captures.markdown:70]: We just completed a project to get rid of the need for ENV vars, so those not being documented is deliberate!

File: docs/captures/captures.markdown [hunk 2]
```diff
@@ -31,11 +31,48 @@ tasks:
...
+## Transparent Proxy task mode
+
+For most local documentation, we recommend starting your API application with an [Optic task](/setup), also known as Command Wrapper mode. This allows Optic to start your project and a new capture session to organize observed requests. Sometimes, this isn't feasible - such as when your API is always deployed in a Serverless architecture and cannot be run locally. In this case, the Optic Transparent Proxy mode will let you configure your downstream and upstream URLs.
+
+### Setting up the Local CLI in Transparent Proxy mode
+
+All configuration items for local capture live in the `optic.yml` file. Create a new task to use the Transparent Proxy mode:
+
+- Create a new task `start-proxy` at the same level of indent as `start` (the default task).
+- `baseUrl` is the URL configuration where the application we're observing should expect to live. When using Optic to observe traffic, Optic adopts this baseUrl to intercept traffic and pass it along to the application under observation. That way, your clients or tests can still run exactly as they have in the past.
```
Comment (JaapRood) [inline docs/captures/captures.markdown:48]: Here we specify `baseUrl`, where further down we mention `inboundUrl`. We should choose the preferred and use that consistently, perhaps listing the other aliases as well. I think `inboundUrl` complements `targetUrl` maybe a bit better than `baseUrl`, but happy with either as the standard.

File: docs/captures/captures.markdown [hunk 3]
```diff
@@ -31,11 +31,48 @@ tasks:
...
+- The local dashboard will be available, with observed diffs.
+- The Optic CLI will listen on localhost at the port defined in `inboundUrl`.
+- Traffic sent to the Optic CLI will be forwarded to the defined `targetUrl` - assure that is already running, or if not, start it before sending any traffic.
+
+### Starting a new Capture Session in Transparent Poxy task mode
+
+When Optic is started in Transparent Proxy task mode, it is not watching the application lifecycle and does not currently capture rebuild events of the application under observation. To start fresh with a new capture session, stop the current Optic proxy and restart with `api run start-proxy`. This will create a new capture session in the Optic dashboard and allow you to observe traffic sent to the most recent build of the application under observation.
+
 ## Ignoring API Paths
 As you use Optic, you will probably find there are some paths you would like Optic to ignore. Examples include `OPTIONS` requests, `.html`, `.js`, `.css` files, or everything in your `/public` folder.
```
[the remark this answers is not in the document]
Comment (LouManglass) [reply docs/captures/captures.markdown:77]: They're still there, just not in the diff view https://github.com/LouManglass/optic-docs/blob/develop/docs/captures/captures.markdown

File: docs/captures/captures.markdown (hunk 2 above)
Comment (LouManglass) [reply docs/captures/captures.markdown:48]: We use baseUrl for command wrapper configurations throughout the documentation. My initial thought was to include the task key alias in examples (which I didn't do in both places here), but this is probably a bigger conversation...should we prefer inboundUrl across the board, and baseUrl is now the less preferred "alias"?

Open to thoughts.

File: docs/captures/captures.markdown (hunk 3 above)
[the remark this answers is not in the document]
Comment (trulyronak) [reply docs/captures/captures.markdown:77]: oh ok, gotcha

[the remark this answers is not in the document]
Comment (LouManglass) [reply docs/captures/captures.markdown:77]: I've done the same thing before :)

**Pull request merged**